server/src/pages/not-found/model.js

```diff
@@ -1,5 +1,5 @@
 exports.createModel = function(req) {
   return {
-    title: "Page Not Found"
+    title: "Page Not Found" // todo l10n: notFoundPageTitle
```

Comment:
Here's another spot where strings are defined in non-view code.

I think the cleanest approach would be to set an l10n ID as the title, then keep all the string substitution in the view code. Thoughts?